### PR TITLE
dax-studio: Add version 2.15.0

### PR DESCRIPTION
**Pull request by adrianocorrea**

### CODE REVIEW

File: bucket/dax-studio.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            "hash": "c6ddd183a095b745c308f83a31081a7d5996f7914684b1028c41ddaa7e45b10a"
+        }
+    },
+    "bin": "DaxStudio.exe",
+    "shortcuts": [
+        [
+            "DaxStudio.exe",
+            "DAX Studio"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/dax-studio.json:18]:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\settings.json\")) { Set-Content \"$dir\\settings.json\" '{}' -Encoding ascii }",
    "shortcuts": [
        [
            "DaxStudio.exe",
            "DAX Studio"
        ],
        [
            "DaxStudio.Checker.exe",
            "DAX Studio Checker"
        ]
    ],
    "persist": [
        "AutoSaveFiles",
        "Log",
        "QueryHistory",
        "settings.json"
    ],
```

File: bucket/dax-studio.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "version": "2.15.0",
+    "description": "Tool for executing and analyzing DAX queries against Microsoft Tabular models",
+    "homepage": "https://daxstudio.org/",
+    "license": "MS-RL",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/DaxStudio/DaxStudio/releases/download/v2.15.0/DaxStudio_2_15_0_portable.zip",
+            "hash": "c6ddd183a095b745c308f83a31081a7d5996f7914684b1028c41ddaa7e45b10a"
+        }
+    },
```
Comment (Ash258) [inline bucket/dax-studio.json:11]:
```suggestion
    "homepage": "https://daxstudio.org",
    "license": "MS-RL",
    "url": "https://github.com/DaxStudio/DaxStudio/releases/download/v2.15.0/DaxStudio_2_15_0_portable.zip",
    "hash": "c6ddd183a095b745c308f83a31081a7d5996f7914684b1028c41ddaa7e45b10a",
```

File: bucket/dax-studio.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    ],
+    "checkver": {
+        "github": "https://github.com/DaxStudio/DaxStudio"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/DaxStudio/DaxStudio/releases/download/v$version/DaxStudio_$underscoreVersion_portable.zip"
+            }
+        }
```
Comment (Ash258) [inline bucket/dax-studio.json:27]:
```suggestion
        "url": "https://github.com/DaxStudio/DaxStudio/releases/download/v$version/DaxStudio_$underscoreVersion_portable.zip"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/dax-studio.json (hunk 3 above)
Comment (adrianocorrea) [reply bucket/dax-studio.json]: Done.

**Review by adrianocorrea — COMMENTED** [review]
Updated formulae.

**Review by Ash258 — CHANGES_REQUESTED** [review]
Always create `bin\.portable` and `.portable` files

https://github.com/DaxStudio/DaxStudio/blob/87f11b343a2e85a24311879d5fe96a090ba55170/src/DaxStudio.Checker/Checker.cs#L713-L718

**Comment by adrianocorrea** [comment]
Sorry, I am not quite sure what is the outstanding issue here. The checker you have highlighted checks if there is a `.portable` file in the bin folder OR in the current folder. The installer is creating the required `.portable` file in the bin folder during the extraction (see folder `.\scoop\apps\dax-studio\current\bin`), which allows the software to work as expected. 

Did I miss something?

**Pull request closed without merging**